README.md

```diff
@@ -160,264 +160,10 @@ Dataset.load(path).then(dataset => {
-
-`
-load(pathOrDescriptor, {basePath, format}={})
-`
-
-There are 3 types of file source we support:
-
-* Local path
-* Remote url
-* Inline data
-
-```javascript
-const data = require('frictionless.js')
-
-const file = data.open('/path/to/file.csv')
-
-const file = data.open('https://example.com/data.xls')
-
-// loading raw data
-const file = data.open({
-  name: 'mydata',
-  data: { // can be any javascript - an object, an array or a string or ...
-    a: 1,
-    b: 2
-  }
-})
-
-// Loading with a descriptor - this allows more fine-grained configuration
-// The descriptor should follow the Frictionless Data Resource model
-// http://specs.frictionlessdata.io/data-resource/
-const file = data.open({
-  // file or url path
-  path: 'https://example.com/data.csv',
-  // a Table Schema - https://specs.frictionlessdata.io/table-schema/
-  schema: {
-    fields: [
-      ...
-    ]
-  }
-  // CSV dialect - https://specs.frictionlessdata.io/csv-dialect/
-  dialect: {
-    // this is tab separated CSV/DSV
-    delimiter: '\t'
-  }
-})
-```
-
-`basePath`: use in cases where you want to create a File with a path that is relative to a base directory / path e.g.
-
-```
-const file = data.open('data.csv', {basePath: '/my/base/path'})
-```
-
-Will open the file: `/my/base/path/data.csv`
-
-This functionality is primarily useful when using Files as part of Datasets where it can be convenient for a File to have a path relative to the directory of the Dataset. (See also Data Package and Data Resource in the Frictionless Data specs).
-
-
-### Files
-
-A single data file - local or remote.
-
-#### load
-
-*DEPRECATED*. Use simple `open`.
-
-#### Metadata
-
-Main metadata is available via the `descriptor`:
-
-```
-file.descriptor
-```
-
-This metadata is a combination of the metadata passed in at File creation (if you created the File with a descriptor object) and auto-inferred information from the File path. This is the info that is auto-inferred:
-
-```
-path: path this was instantiated with - may not be same as file.path (depending on basePath)
-pathType: remote | local
-name:   file name (without extension)
-format: the extension
-mediatype: mimetype based on file name and extension
-```
-
-In addition to this metadata there are certain properties which are computed on demand:
-
-```javascript
-// the full path to the file (using basepath)
-const path = file.path
-
-const size = file.size
-
-// md5 hash of the file
-const hash = file.hash()
-
-// sha256 hash of the file
-const hash256 = file.hash(hashType='sha256')
-
-// file encoding
-const encoding = file.encoding
-```
-
-**Note**: size, hash are not available for remote Files (those created from urls).
-
-
-#### stream
-
-`stream()`
-
-Get readable stream
-
-@returns Promise with readable stream object on resolve
-
-#### buffer
-
-`File.buffer`
-
-Get this file as a buffer (async)
-
-@returns: promise which resolves to the buffer
+See API documentation [here](./docs)
```

Comment:
A lot of headings have been removed from API section, so the table of content above needs to be updated.

Reply:
Okay, thanks for noting this.